data/Harry_Potter_Characters.tsv

```diff
@@ -52,6 +52,7 @@ Ginny Weasley,Daughter of Molly and Arthur.,5,5,7,6
 Molly Weasley,Wife of Arthur and mother of the Weasleys.,4,5,8,9
 Percy Weasley,Third son of Arthur and Molly. He is a Gryffindor prefect.,5,4,8,4
 Ron Weasley,Harry's best friend.,6,6,6,7
-Rachel Richards, Hopeless Harry Potter Fan and Co-Creator of this Card Game,9,10,0,5
-Wafiakmal Mitah, Hopelessly Hopeless Harry Potter Fanboy Extraordinaire and Co-Creator of this Card Game,10,9,0,5
-Eric Rios, The One Guy Who Wanted this Card Game to be Anything but Harry Potter. Forced Co-Creator.,0,0,10,10
\ No newline at end of file
+Rachel Richards,Hopeless Harry Potter Fan and Co-Creator of this Card Game,9,10,0,5
+Wafiakmal Mitah,Hopelessly Hopeless Harry Potter Fanboy Extraordinaire and Co-Creator of this Card Game,10,9,0,5
+Eric Rios,The One Guy Who Wanted this Card Game to be Anything but Harry Potter. Forced Co-Creator.,0,0,10,10
+Patrick Wang,The newest professor of Defense Against the Dark Arts,7,8,10,5
```

Comment:
This is beautiful, we can tell him to play the game until he finds that surprise!!!!!!!!!!!!!!!!!!!!!!!!!!!1